test/OpenFeature.Contrib.Providers.Flagsmith.Test/FlagsmithProviderTest.cs

```diff
@@ -65,7 +65,7 @@ public async Task GetValue_ForEnabledFeatureWithEvaluationContext_ReturnCorrectV
             var date = DateTime.Now;
             flags.GetFeatureValue("example-feature").Returns("true");
             flags.IsFeatureEnabled("example-feature").Returns(true);
-            flagsmithClient.GetIdentityFlags("233", Arg.Is<List<ITrait>>(x => x.Count == 6 && x.Any(c => c.GetTraitKey() == "key1"))).Returns(flags);
+            flagsmithClient.GetIdentityFlags("233", Arg.Is<List<ITrait>>(x => x.Count > 6 && x.Any(c => c.GetTraitKey() == "key1"))).Returns(flags);
```

Comment:
This is perhaps the only slightly surprising change here. We include the targeting key in the props now, which makes the count here 1 larger than before.